swatch-billable-usage/deploy/clowdapp.yaml

```diff
@@ -232,6 +234,32 @@ objects:
               cpu: ${CURL_CRON_CPU_LIMIT}
               memory: ${CURL_CRON_MEMORY_LIMIT}
 
+      - name: retry-remittances
+        schedule: ${RETRY_REMITTANCE_SCHEDULE}
+        activeDeadlineSeconds: 1800
+        successfulJobsHistoryLimit: 2
+        restartPolicy: Never
+        podSpec:
+          image: ${CURL_CRON_IMAGE}:${CURL_CRON_IMAGE_TAG}
+          command:
+            - /usr/bin/bash
+            - -c
+            - >
+              /usr/bin/curl --fail -H "Origin: https://swatch-bilable-usage-service.redhat.com" -H "x-rh-swatch-psk: ${SWATCH_SELF_PSK}" -X POST "http://swatch-billable-usage-service:8000/api/swatch-billable-usage/internal/rpc/remittance/processRetries"
```

Comment:
There's a typo in the `Origin` header here I think?  Should be `swatch-billable-usage-service.redhat.com` (with 2 `l`'s in `billable`)?

Reply:
Good spotted! I've also fixed it in the rest of the cronjobs.